src/AccessibilityInsights.SharedUx/FileIssue/IssueReporter.cs

```diff
@@ -39,6 +63,11 @@ public static Task RestoreConfigurationAsync(string serializedConfig)
 
         public static IIssueResult FileIssueAsync(IssueInformation issueInformation)
         {
+            if (TestControlledFileIssueAsync != null)
```

Comment:
It might be good to use braces elsewhere too for consistency on one-line if-statements (TestControlled-* statements above)

Reply:
Added